### PR TITLE
fix #190

### LINKED ISSUE #190: Gracious shutdown of snare
Currently, on quitting snare throws a lot of KeyboardInteuppt exception which looks terrible. I think it can be made to quit graciously just like tanner server.

### PR DESCRIPTION
**Pull request by teeann**
Use lower-level Process API to allow gracious shutdown.

### CODE REVIEW

File: bin/snare [hunk 1]
```diff
@@ -202,8 +201,6 @@ if __name__ == '__main__':
         loop.run_forever()
     except (KeyboardInterrupt, TypeError) as e:
         loop.run_until_complete(app.stop())
+        compare_version_proc.terminate()
         print(e)
```
Comment (afeena) [inline bin/snare]: Could you please remove this print? :) I spend some time trying to understand why I still have error in console :D

**Comment by Aniket21mathur** [comment]
@afeena @gaochua I tested the patch, I still got a traceback

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/Snare-0.3.0-py3.6.egg/EGG-INFO/scripts/snare", line 104, in compare_version_info
KeyboardInterrupt
```
Might be because the process is interrupted in between of a timeout.

**Comment by afeena** [comment]
@Aniket21mathur thanks for checking this. 
@teeann please verify

**Comment by afeena** [comment]
Needs changes and testing, please reopen if needed

**Pull request closed without merging**